packages/service/package.json

```diff
@@ -78,6 +78,7 @@
     "es-main": "^1.3.0",
     "execa": "^8.0.1",
     "fastify": "^5.0.0",
+    "fastify-metrics": "^12.0.0",
```

Comment:
should not be needed

Reply:
I think it's still used in a platformatic service

Reply:
https://github.com/platformatic/platformatic/blob/f85e6a61c8039b5820ea1fcbc3e12168de483ae0/packages/service/lib/plugins/metrics.js#L21